wordpress-mobile/WordPress-Android
# [Gutenberg] Use Hermes bytecode

This PR is another attempt after https://github.com/wordpress-mobile/WordPress-Android/pull/11436

If it proves more successful I will close the other one.

Gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2780

**No need to merge this PR per se as it's practically only bumps the gb-mobile hash; the improvements will be included anyway when the gb-mobile side PRs merge.**

To test:
No special testing other than all operations should look and feel faster.
One easy to spot improvement (I guess) is to start a new post and notice how much faster the editor comes online.

PR submission checklist:
